src/scss/config/_scale.scss

```diff
@@ -34,7 +34,7 @@ $font-sizes: (
   'h1': '#xlarge',
   'h2': '#large',
   'h3': '#medium',
-  'code': 0.9em,
+  'code': 0.8rem,
```

Comment:
@mirisuzanne this seems to even out the size of code since before hte paragraph code was pretty large and example code was smaller.